sunpy/map/sources/rhessi.py

```diff
@@ -41,11 +39,8 @@ def __init__(self, data, header, **kwargs):
 
         self.meta['waveunit'] = 'keV'
         self.meta['wavelnth'] = [self.meta['energy_l'], self.meta['energy_h']]
-        self.cmap = plt.get_cmap('jet')
-
-    @property
-    def measurement(self):
-        return u.Quantity([self.meta['energy_l'], self.meta['energy_h']], self.meta['waveunit'])
+        self.plot_settings['cmap'] = cm.get_cmap('rhessi')
+        self._name = "RHESSI {measure[0]:.0f} - {measure[1]:.0f}".format(measure=self.measurement)
```

Comment:
Um, how does this get used now that you've changed `Map.name` to not use `self._name`?